cirras-underwriting-war/cirras-underwriting-war/src/main/angular/src/app/components/verified-yield/amendment/verified-yield-amendment.component.html

```diff
@@ -0,0 +1,93 @@
+        <mat-form-field appearance="fill">
+            <input 
+                type="text" 
+                matInput 
+                formControlName="cropCommodityCtrl" 
+                [matAutocomplete]="autoCropCommodityId"
+                (focus)="commoditiesFocus()"
+                (keyup)="searchCommodity($event.target.value)">
+            <mat-autocomplete #autoCropCommodityId="matAutocomplete" hideSingleSelectionIndicator="true"
+                [displayWith]="displayCommoditiesFn"
+                (optionSelected)="updateCommodity()">
+            <mat-option *ngFor="let option of filteredCropCommodityOptions" [value]="option">
+                {{ option.cropCommodityName | titlecase }}
+            </mat-option>
+            </mat-autocomplete>
+        </mat-form-field>  
+        <mat-icon style="padding-right:10px; font-size:18px; margin-top:6px;" matTooltip="Editable field">edit</mat-icon>
+    </div>
+
+    <div *ngIf="amendmentFormGroup.value.deletedByUserInd != true" class="inventory-fill number border-right border-bottom">
+        <mat-form-field appearance="fill">
+            <input matInput matInputCommified formControlName="yieldPerAcre" maxlength="10"
+                (keypress)="numberOnly($event)" (blur)="updateYieldPerAcre()" />
+        </mat-form-field>
+        <mat-icon style="padding-right: 10px; font-size: 18px; margin-top: 6px;"
+            matTooltip="Editable field">edit</mat-icon>
+    </div>
+
+    <div *ngIf="amendmentFormGroup.value.deletedByUserInd != true" class="inventory-fill number border-right border-bottom">
+        <mat-form-field appearance="fill">
+            <input matInput matInputCommified formControlName="acres" maxlength="10"
+                (keypress)="numberOnly($event)" (blur)="updateAcres()" />
+        </mat-form-field>
+        <mat-icon style="padding-right:10px; font-size:18px; margin-top:6px;"
+            matTooltip="Editable field">edit</mat-icon>
+    </div> 
+    
+    <div *ngIf="amendmentFormGroup.value.deletedByUserInd != true" class="border-right border-bottom center">
+        <mat-form-field appearance="fill"  style="width: 100%;" class="inventory-fill">
+            <textarea matInput formControlName="rationale" 
+                cdkTextareaAutosize #autosize="cdkTextareaAutosize"
+                cdkAutosizeMinRows="1" cdkAutosizeMaxRows="8" 
+                maxlength="200" (blur)="updateRationale()">
+                {{ amendmentFormGroup.value.rationale }}
+            </textarea>
+        </mat-form-field>
+        <mat-icon style="padding-right:10px; font-size:18px; margin-top:6px;"
+            matTooltip="Editable field">edit</mat-icon>
+    </div>
+
+    <div *ngIf="amendmentFormGroup.value.deletedByUserInd != true" class="readonly border-right border-bottom center">
+        <mat-icon *ngIf="securityUtilService.canEditVerifiedYield()"
+            matTooltip="Clear editable fields for amendment." 
```

Comment:
I think we want to say that the amendment will be deleted and not just clear the input fields.